repos/fdbt-site/src/design/modules/_cards_alternative.scss

```diff
@@ -0,0 +1,35 @@
+.card {
```

Comment:
What's the difference between cards and cards_alternative?

Reply:
`_cards_alternative.scss` is the one where the actions are at the bottom. It isn't included in our `_main.scss` so it doesn't get pulled in, but it is there if we even want to bring it in.

Reply:
Ah cool - do we want to remove this before we merge to develop now to avoid confusion?